### PR TITLE
Feature/bella mama with instructions

### PR DESCRIPTION
**Pull request by brendawafula**

### CODE REVIEW

**Comment by michaelclapham** [comment]
First instruction should be
Play with the words and _sounds_

**Comment by michaelclapham** [comment]
The first instruction should 
Play with the words and _sounds_

**Comment by michaelclapham** [comment]
Where they are speaking the lyrics but it's not specifically an instruction from 0:02 to 0:26 should be not instructions, but labelled as person 1 or person 2 so we can see what each person is saying. Similar to how the lyrics are displayed after 0:28.

**Comment by michaelclapham** [comment]
All the times you've written Bella mama
eeeeeeeeh
the lyric is actually
Bella mama **_heeey_**

**Comment by michaelclapham** [comment]
The instruction at 1:17 that's currently "Split the group into four sing all together"
should be two instructions
3. Split the group in two
4. Sing all together

**Comment by michaelclapham** [comment]
At about 2:01 and 2:04 (2:03.5 if you want to be precise) we should see the lyric with 6 Bella mama's. At 2:01 we should show it for person 1, and 2:04 for person 2.

File: src/app/lyrics/songs/bella-mama-with-instructions.ts [hunk 1]
```diff
@@ -122,7 +122,7 @@ export const Bella_Mama_Instructions: Song = {
             "endTimeSecond": "17"
         },
         {
-            "lyric": "Bella mama eeeeeeh",
+            "lyric": "Bella mama heeeeeey",
             "person": "2",
             "isInstructions": "TRUE",
```
Comment (maxwellfundi) [inline src/app/lyrics/songs/bella-mama-with-instructions.ts:127]: @brendawafula  we should change to not being instructions,

Comment (maxwellfundi) [inline src/app/lyrics/songs/bella-mama-with-instructions.ts:127]:
```suggestion
            "isInstructions": "FALSE",
```

**Comment by maxwellfundi** [comment]
@brendawafula  This needs a little more work to be okay. See my code suggestions on the isInstructions
You also need to fix some issues raised by Mike and check on the instruction at 1.20 its - Split the group in to two no into..

File: src/app/lyrics/songs/bella-mama-with-instructions.ts [hunk 2]
```diff
@@ -131,7 +131,7 @@ export const Bella_Mama_Instructions: Song = {
             "endTimeSecond": "19"
         },
         {
-            "lyric": "Bella mama eeeeeeh",
+            "lyric": "Bella mama heeeeeey",
             "person": "1",
             "isInstructions": "TRUE",
```
Comment (maxwellfundi) [inline src/app/lyrics/songs/bella-mama-with-instructions.ts:136]:
```suggestion
            "isInstructions": "FALSE",
```

**Comment by maxwellfundi** [comment]
@brendawafula  I have made a few changes here and merging this. it looks good except for just one thing  that i would like you to do in the morning- **At about 2:01 and 2:04 (2:03.5 if you want to be precise) we should see the lyric with 6 Bella mama's. At 2:01 we should show it for person 1, and 2:04 for person 2.**

**Pull request merged**